### ISSUE TITLE
Cmus freezes on ape file.

### ISSUE THREAD

**Issue by castilma** [cmus/cmus#1174]
I remember playing [this](https://seafile.zfn.uni-bremen.de/f/fdc1ba6525674014995f/?dl=1) file in the past with cmus, but now it freezes with one core busy. ffplay plays it without problem. SIGTERM and SIGHUP don't work, but SIGQUIT terminates cmus.
If you strip the last byte, cmus just skips the file:
`head -c  345219 345220.ape > 345219.ape`

```
ffplay version n5.0 Copyright (c) 2003-2022 the FFmpeg developers
  built with gcc 11.2.0 (GCC)
  configuration: --prefix=/usr --disable-debug --disable-static --disable-stripping --enable-amf --enable-avisynth --enable-cuda-llvm --enable-lto --enable-fontconfig --enable-gmp --enable-gnutls --enable-gpl --enable-ladspa --enable-libaom --enable-libass --enable-libbluray --enable-libdav1d --enable-libdrm --enable-libfreetype --enable-libfribidi --enable-libgsm --enable-libiec61883 --enable-libjack --enable-libmfx --enable-libmodplug --enable-libmp3lame --enable-libopencore_amrnb --enable-libopencore_amrwb --enable-libopenjpeg --enable-libopus --enable-libpulse --enable-librav1e --enable-librsvg --enable-libsoxr --enable-libspeex --enable-libsrt --enable-libssh --enable-libsvtav1 --enable-libtheora --enable-libv4l2 --enable-libvidstab --enable-libvmaf --enable-libvorbis --enable-libvpx --enable-libwebp --enable-libx264 --enable-libx265 --enable-libxcb --enable-libxml2 --enable-libxvid --enable-libzimg --enable-nvdec --enable-nvenc --enable-shared --enable-version3
```

---

1) `uname -a`:
```
Linux T500 5.16.7-artix1-1 #1 SMP PREEMPT Sun, 06 Feb 2022 19:00:59 +0000 x86_64 GNU/Linux
```

2) `cmus --version`:
```
cmus v2.9.1
Copyright 2004-2006 Timo Hirvonen
Copyright 2008-2016 Various Authors
```

3) `cmus --plugins`:
```
Input Plugins: /usr/lib/cmus/ip
  mad:
    Priority: 55
    File Types: mp3 mp2
    MIME Types: audio/mpeg audio/x-mp3 audio/x-mpeg
  mpc:
    Priority: 50
    File Types: mpc mpp mp+
    MIME Types: audio/x-musepack
  modplug:
    Priority: 50
    File Types: mod s3m xm it 669 amf ams dbm dmf dsm far mdl med mtm okt ptm stm ult umx mt2 psm
    MIME Types:
  cue:
    Priority: 50
    File Types:
    MIME Types: application/x-cue
  cdio:
    Priority: 50
    File Types:
    MIME Types: x-content/audio-cdda
  vorbis:
    Priority: 50
    File Types: ogg oga ogx
    MIME Types: application/ogg audio/x-ogg
  wav:
    Priority: 50
    File Types: wav
    MIME Types:
  flac:
    Priority: 50
    File Types: flac fla
    MIME Types:
  wavpack:
    Priority: 50
    File Types: wv
    MIME Types: audio/x-wavpack
  aac:
    Priority: 50
    File Types: aac
    MIME Types: audio/aac audio/aacp
  opus:
    Priority: 50
    File Types: opus
    MIME Types:
  ffmpeg:
    Priority: 30
    File Types: aa aac ac3 aif aifc aiff ape au fla flac m4a m4b mka mkv mp+ mp2 mp3 mp4 mpc mpp ogg opus shn tak tta wav webm wma wv
    MIME Types:

Output Plugins: /usr/lib/cmus/op
  pulse
  alsa
  oss
  jack
  ao
```

**Comment by sylphenix** [comment]
I have the same problem. Cmus can't play any ape file on my machine. I tried on Slackware 15 and Debian 11.

**Comment by dryleev** [comment]
Looks like a duplicate of #812 which has been fixed in `master`.

**Comment by timsofteng** [comment]
Still doesn't work on 2.9.1 in alpine Linux.

**Comment by ledufff** [comment]
this seems to have been fixed for cmus 2.10.0 and later IME.

debian testing and sid
and archlinux

**Comment by castilma** [comment]
v2.10.0 works for me
